scripts/js/lib/api/Pkg.ts

```diff
@@ -86,7 +92,6 @@ export class Pkg {
       return new Pkg({
         ...args,
         title: "Qiskit SDK",
-        name: "qiskit",
```

Comment:
No need to set this. `name` was already set in `..args`. It was validated that it equals `qiskit` via the `if` statement right above.